### ISSUE TITLE
Second row in empty editor in IE10

### ISSUE THREAD

**Issue by jeserkin** [fraywing/textAngular#903]
Recently found, that in `IE10` when I open empty editor and click in the middle of it, then cursor appears on second row, not on the first one.
![textarea](https://cloud.githubusercontent.com/assets/183512/10041604/e401beca-61ed-11e5-8c9d-3485aa50f7a1.png)
`IE9` and `IE11` seem to work fine.

Can it be fixed or handled by any means?

**Comment by pbassut** [comment]
I'm creating a VM to test this since I use osx. Stay tunned

**Comment by pbassut** [comment]
Ehh, looks like IE9 can only be install on vista. Does that happen on IE10?

**Comment by jeserkin** [comment]
Yes and `IE9` can be installed on Windows 7. I just need to rollback the update if it did automatically install itself.

**Comment by pbassut** [comment]
Does this happen on [textAngular.com](http://textAngular.com)?

**Comment by jeserkin** [comment]
Tried. Seems that it is only in our app. We use 1.3.7 version. Maybe there was some bug at that time.

**Comment by chrisknepper** [comment]
@jeserkin This indeed appears to be a bug in the textAngular 1.4.6, specific to IE < 11. I experience it in IE10.

This does not occur on the textAngular website. It seems to be specific to when the textarea is empty (or just has a placeholder) upon page load, which the instance on the textAngular website does not.

I fixed it in my project by changing line 155 in `src/taBind.js`, removing the `<br>`. Also, these default value strings appear to depend on the browser version, but IE10 is left out. The instances where it checks `_browserDetect.ie >= 11` should probably be `_browserDetect.ie >= 10`, though the extra line in IE10 seems to be fixed by removing the `<br>` alone.

I may be able to create a pull request for this.

Also @pbassut, tip for us Mac users: [free Windows VMs from Microsoft for debugging IE issues including IE8, IE9, IE10, and IE11 on Windows 7](https://dev.modern.ie/tools/vms/mac/)

**Comment by pbassut** [comment]
@chrisknepper Oh, that's a great tip. Thank you!
I'm downloading the image to investigate a bit more.
Stay tuned guys...

**Comment by jeserkin** [comment]
@pbassut any news?

**Comment by jeserkin** [comment]
@pbassut still no news?

**Comment by pbassut** [comment]
Sorry, I've been overwhelmed with my new job. I'm trying to find some free time to tackle this.

**Comment by jeserkin** [comment]
Seems it will takes too long to wait. Will look into it myself.

**Comment by jeserkin** [comment]
As it seems the main issue is with ``_browserDetect`` object, that is initialized on load. It doesn't set IE version correctly and therefore further code isn't executed correctly. Will try to correct the initialization process.

**Comment by jeserkin** [comment]
So I've combined two IE ver. recognition solutions together. That what came to be
```javascript
var _browserDetect = {
    ie: (function() {
        var undef,
                v = 3,
                div = document.createElement('div'),
                all = div.getElementsByTagName('i');

        while (
                div.innerHTML = '<!--[if gt IE ' + (++v) + ']><i></i><![endif]-->',
                        all[0]
                );

        var result = v > 4 ? v : undef;

        if (typeof result === 'undefined') {
            result = (function() {
                var ua = window.navigator.userAgent,
                        msie = ua.indexOf('MSIE ');
                if (msie > 0) {
                    // IE 10 or older => return version number
                    return parseInt(ua.substring(msie + 5, ua.indexOf('.', msie)), 10);
                }

                var trident = ua.indexOf('Trident/');
                if (trident > 0) {
                    // IE 11 => return version number
                    var rv = ua.indexOf('rv:');
                    return parseInt(ua.substring(rv + 3, ua.indexOf('.', rv)), 10);
                }

                var edge = ua.indexOf('Edge/');
                if (edge > 0) {
                    // Edge (IE 12+) => return version number
                    return parseInt(ua.substring(edge + 5, ua.indexOf('.', edge)), 10);
                }
            })();
        }

        return result;
    }()),
    webkit: /AppleWebKit\/([\d.]+)/i.test(navigator.userAgent)
};
```
